src/validators/__stringifyHelpers.ts

```diff
@@ -0,0 +1,27 @@
+import type { SomeSchema } from '../types';
+
+export const typeToPrint = (str: string) => '≫' + str + '≪';
```

Comment:
I'm afraid that this symbol's (`≫`) display depends on the installed font on the  machine that will run our library, so I think the display of this symbol may look ugly or not be displayed at all.

Reply:
Any examples?

Reply:
![image](https://user-images.githubusercontent.com/47081011/105464907-8cac1300-5c92-11eb-9ff3-1161b91bb243.png)